citrination_client/client.py

```diff
@@ -151,6 +151,31 @@ def _get_list(values):
         else:
             return [values]
 
+    def predict(self, model_name, candidates):
+        """
+        Predict endpoint
+        :param model_name: The model path
+        :param candidates: A list of candidates
+        :return: list of predicted candidates as a map {property: [value, uncertainty]}
+        """
+
+        # If a single candidate is passed, wrap in a list for the user
+        if not isinstance(candidates, list):
+            candidates = [candidates]
+
+        url = self._get_predict_url(model_name)
+        body = pif.dumps(
+            {"predictionRequest": {"predictionSource": "scalar", "usePrior": True, "candidates": candidates}}
+        )
+
+        response = requests.post(url, data=body, headers=self.headers)
+        if response.status_code == requests.codes.ok:
+            return response.json()['candidates']
```

Comment:
Do our users care about an HTTP response or the prediction results? I don't really know what your plans were for this, but I would expect the data, not a Response object.